install/install.sh

```
@@ -0,0 +1,184 @@
+    local file="$3"
+
+    if grep -q "^$variable=" "$file"; then
+        sudo sed -i "s|^$variable=.*|$variable=$value|" "$file"
+    else
+        echo "$variable=$value" >> "$file"
+    fi
+}
+
+clean_dir() {
+    local repo_url="$1"
+    local dest_dir=$(basename -s .git "$repo_url")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)